src/main/java/brave/opentracing/BraveSpanBuilder.java

```diff
@@ -105,10 +111,32 @@
     return this;
   }
 
+  @Override
+  public Span start() {
+    return startManual();
+  }
+
+  @Override
+  public ActiveSpan startActive() {
+    if (!ignoreActiveSpan) {
```

Comment:
That seems reasonable, but I think that having the library be responsible for keeping track of the active span greatly simplifies the application code for most use cases. I came into all of this as an application developer instrumenting services, and from that perspective I really want that stuff consolidated in one place.